src/pages/ReimbursementAccount/BankInfo/BankInfo.tsx

```diff
@@ -97,14 +90,14 @@ function BankInfo({reimbursementAccount, reimbursementAccountDraft, plaidLinkTok
 
     const handleBackButtonPress = () => {
         if (screenIndex === 0) {
-            // TODO replace it with navigation to ReimbursementAccountPage once base is updated
```

Comment:
Why it was removed?

Reply:
just cleanup, ReimbursementAccountPage wont be updated anytime soon so this TODO is unnecessary